test/runAnalysis/runAnalysis.py

```diff
@@ -358,10 +377,27 @@ def main():
 				if not args.do2016 and not args.do2018 and not args.ci2017 and not args.data and not args.resolution and args.electrons:
 					print "trying"
 					cmssw_tmp = cmssw_tmp.replace('mc_2017',dataset_name)
+
+                                # write add filename information
+				if args.add2016:
+                                	os.system('dasgoclient -query="file dataset=%s | grep file.name" > myfiles.txt'%dataset)
+                                	with open('myfiles.txt') as fin:
+                                        	content = fin.readlines()
+                                	content = [x.strip() for x in content]
+					print content[0]
+					fin.close()
+					cmssw_tmp = cmssw_tmp.replace('/store/data/Run2017F/DoubleEG/MINIAOD/17Nov2017-v1/50000/00105BAD-63E0-E711-8640-02163E0146C5.root', content[0])
+
 				#print getFilterSnippet(dataset_name)
 				open('cmssw_cfg.py', 'wt').write(cmssw_tmp)
             			if args.submit:
-                			os.system('crab submit -c crabConfig.py')
+                			#os.system('crab submit -c crabConfig.py')
+					os.system('cmsRun cmssw_cfg.py')
+					fnamere = "dileptonAna_%s%s.root"%(prefix, dataset_name)
+                                        if "muon" in fnamere: fnamere = fnamere.replace("LL", "2Mu")
+                                        else: fnamere = fnamere.replace("LL", "2E")
+                                        os.system('mv zp2mu_histos.root %s'%fnamere)
+                                        #print "Test passed for: " + dataset_name
```

Comment:
I'm pretty sure all this stuff here breaks the submission to CRAB! I think it is a neat implementation on how to run on a sample locally, but that should be an extra feature and not break existing functionality!